public/js/profile.js

```diff
@@ -0,0 +1,68 @@
+/**
+* Submits the new updates for the profile
+*/
+$('#editForm').submit(function(evt) {
+    evt.preventDefault();
+
+    if ($('#passwd').val() === $('#confirmpasswd').val()) {
+        var id = $('#userId').html();
+        editProfile(id);
+    } else {
+        failSnackbar('Passwords do not match');
+    }
+});
+
+/**
+* Changes the view to the editing page
+*
+* @method enableEdit
+*/
+var enableEdit = function() {
+    $('#viewForm').addClass('hidden');
+    $('#editForm').removeClass('hidden');
+    $('#cancelButton').removeClass('hidden');
+}
+
+/**
+* Changes the view to the display page
+*
+* @method disableEdit
+*/
+var disableEdit = function() {
+    location.reload();
+}
+
+var editProfile = function(id) {
+    var fields = $('#editForm').serializeArray();
+    var user = {
+        originalID: id
```

Comment:
This is not needed anymore, please remove it. Also we can for all form fields down here, don't we just need a few of them.